lib/matplotlib/path.py

```diff
@@ -642,6 +642,47 @@ def intersects_bbox(self, bbox, filled=True):
         return _path.path_intersects_rectangle(
             self, bbox.x0, bbox.y0, bbox.x1, bbox.y1, filled)
 
+    def signed_area(self, **kwargs):
+        """
+        Get signed area of the filled path.
+
+        All sub paths are treated as if they had been closed. That is, if there
+        is a MOVETO without a preceding CLOSEPOLY, one is added.
+
+        If the path is made up of multiple components that overlap, the
+        overlapping area is multiply counted.
+
+        Returns
+        -------
+        float
+            The (signed) enclosed area of the path.
```

Comment:
I think 1) I would remove the parenthesis, 2) somewhere you should mention whether clockwise or counterclockwise is positive, 3) not everyone knows what a signed area with, perhaps put a very simple note "e.g. a figure eight where the top loop is clockwise and the bottom loop counterclockwise will have signed area zero, because etc."

Reply:
Let me know if the new version looks good!